image/cli/mascli/functions/configure_ocp_for_mirror

```diff
@@ -11,9 +11,9 @@ Where ${COLOR_YELLOW}specified${TEXT_RESET} each option may also be defined by s
 When no options are specified on the command line, interactive-mode will be enabled by default.
 
 Private Registry Details (Required):
-  -H, --host ${COLOR_YELLOW}REGISTRY_PRIVATE_HOST${TEXT_RESET}        Hostname of the private registry
-  -P, --port ${COLOR_YELLOW}REGISTRY_PRIVATE_PORT${TEXT_RESET}        Port number for the private registry
-  -c, --ca-file ${COLOR_YELLOW}REGISTRY_PRIVATE_CA_FILE${TEXT_RESET}  Location of the CA file for the private registry
+  -H, --host ${COLOR_YELLOW}REGISTRY_PUBLIC_HOST${TEXT_RESET}        Hostname of the private registry
```

Comment:
Why change this?  Public and private are supported for use case where customer has a registry bridging multiple networks.

- mirror from the internet connected host to the public host of the registry
- configure OCP on the disconnected network to use the private host of the registry

In some cases there may be only one network and private == public, but this is designd to support more advanced scenarios too; so I think this needs to be changed back unless I'm missing something here?